main.cpp

```diff
@@ -27,7 +27,7 @@ int detect_codes(zbar::ImageScanner &scanner, cv::Mat &frame) {
 #endif
 
   cv::Mat grey;
-  cv::cvtColor(frame, grey, CV_BGR2GRAY);
+  cv::cvtColor(frame, grey, cv::COLOR_BGR2GRAY);
```

Comment:
```suggestion
#ifdef CV_BGR2GRAY
  cv::cvtColor(frame, grey, CV_BGR2GRAY);
#else
  cv::cvtColor(frame, grey, cv::COLOR_BGR2GRAY);
#endif
```
to not break OpenCV 3.* compatibility